tests/CMakeLists.txt

```
@@ -0,0 +1,23 @@
+enable_testing()
```

```suggestion
enable_testing()
```

This line is harmless but unneeded. It's generally best to just put `include(CTest)` in the top-level `CMakeLists.txt` to describe the whole project as being "test capable". It's a property of the project as a whole.